command-parser.js

```diff
@@ -30,6 +30,8 @@ const MESSAGE_COOLDOWN = 5 * 60 * 1000;
 
 const MAX_PARSE_RECURSION = 10;
 
+const VALID_COMMAND_TOKENS = ['/', '!'];
```

Comment:
Seems weird for this to be a const when support for the individual tokens are hardcoded.